### DISCUSSION TITLE
CSS multiple ampersand support

### DISCUSSION THREAD

**Discussion by mpech** [biomejs/biome#3348] [Ideas]
Not really part of the spec (as far as I know)

but lightningcss supports it, esbuild not really but does not fail on it

[playground](https://biomejs.dev/playground/?files.main.css=LgB0AGUAcwB0ACAAewAKACAAIAAmACYAIAB7AAoAIAAgACAAIABjAG8AbABvAHIAOgAgAHAAaQBuAGsAOwAKACAAIAB9AAoAIAAgAGQAaQB2ACAAewAKACAAIAAgACAAJgAmACYAIAB7AAoAIAAgACAAIAAgACAAYwBvAGwAbwByADoAIABwAGkAbgBrADsACgAgACAAIAAgAH0ACgAgACAAfQAKAH0A)

example of translation (if it helps)
lightningcss (1.25.1):
```
.vr-ZQJbCW-test.vr-ZQJbCW-test {
  color: pink;
}
.vr-ZQJbCW-test div:is(.vr-ZQJbCW-test div):is(.vr-ZQJbCW-test div) {
  color: pink;
}

```
esbuild (0.23.0):
```
.src_test {
  & {
    color: pink;
  }
  div {
    & {
      color: pink;
    }
  }
}

``` 

it would be really nice if biome can support &+ syntax, as copying `:is()` is really cumbersome

**Comment by denbezrukov** [comment]
I think it's a bug.
Thank you for the report!
